redisvl/extensions/session_manager/base_session.py

```diff
@@ -0,0 +1,173 @@
+from typing import Any, Dict, List, Optional, Union
+
+from redis import Redis
+
+
+class BaseSessionManager:
+    id_field_name: str = "id_field"
+    role_field_name: str = "role"
+    content_field_name: str = "content"
+    tool_field_name: str = "tool_call_id"
+    timestamp_field_name: str = "timestamp"
+
+    def __init__(
+        self,
+        name: str,
+        session_tag: str,
+        user_tag: str,
+    ):
+        """Initialize session memory with index
+
+        Session Manager stores the current and previous user text prompts and
+        LLM responses to allow for enriching future prompts with session
+        context. Session history is stored in prompt:response pairs referred to
```

Comment:
I believe this was made more flexible to not just store in prompt:response pairs? Might just be update to comment.